### DISCUSSION TITLE
Question on WebView Security

### DISCUSSION THREAD

**Discussion by togithubdeepak** [MicrosoftEdge/WebView2Feedback#3919] [Q&A]
if the User Directory is the same, two different processes could share the same edge-view instance. and allows a process to filter other's requests and responses. is there a way to lock down the webview to only select processes?

**Comment by LiangTheDev** [comment]
Use [ExclusiveUserDataFolderAccess ](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2experimentalenvironmentoptions?view=webview2-1.0.1018-prerelease#put_exclusiveuserdatafolderaccess)option. It is currently an experimental API.
It will only allow one process though. Multiple selected processes are not possible.
